src/cities/CityConfig.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+
+class CityConfig {
+
+  cmsName: string
+  hostName: string
+  formsEnabled: boolean
+  title: string
+  logo: string
+  impressumUrl: string
+
```

Comment:
Imo an accent color would be nice for styling.

Reply:
We can create a new task for this. But I'd say we wait if we really need this. As soon as the first city wants a specific theme we can do this.